packages/components/src/slider/styles.ts

```diff
@@ -0,0 +1,194 @@
+/**
+ * External dependencies
+ */
+import { css } from '@emotion/react';
+import type { CSSProperties } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { COLORS, CONFIG } from '../utils';
+import { flow } from '../utils/flow';
+import { SliderColors } from './types';
+
+const getBoxShadowStyle = (
+	color: CSSProperties[ 'color' ] = COLORS.admin.theme
+) => {
+	return flow(
```

Comment:
I wonder if the `flow` utility is really necessary — could we just interpolate strings instead?

Reply:
Agreed. I've removed the `flow` util in https://github.com/WordPress/gutenberg/pull/42315/commits/daf1515529a287afa74dbf833492fe4b81ebc3d2